src/webServer/webServer.ts

```diff
@@ -0,0 +1,193 @@
+/*@internal*/
+namespace ts.server {
+    export interface HostWithPostMessage {

```

Comment:
I guess I would naively have expected this type to have a `postMessage` member.

Reply:
Sorry for naming i was playing with naming and i will make it `HostWithWriteMessage`